packages/app/src/utils/tenderSwapHooks.ts

```diff
@@ -60,6 +61,10 @@ export const useCalculateRemoveLiquidityOneToken = (
   return result?.value?.[0] ?? constants.Zero;
 };
 
+export const getExecutionPrice = (receiveAmount: BigNumber, sendAmount: string) => {
```

Comment:
small question, shouldn't this be `useExecutionPrice`?

Reply:
no, because it's not a hook, just a function